services/jetbrains/jetbrains-downloads.tester.js

```diff
@@ -17,25 +17,21 @@ t.create('downloads (user friendly plugin id)')
 
 t.create('downloads')
   .get('/9435.json')
-  .intercept(
-    nock =>
-      nock('https://plugins.jetbrains.com')
-        .get('/plugins/list?pluginId=9435')
-        .reply(
-          200,
-          `<?xml version="1.0" encoding="UTF-8"?>
-            <plugin-repository>
-              <category name="Code editing">
-                <idea-plugin downloads="2" size="13159" date="1485601807000" url=""></idea-plugin>
-              </category>
-            </plugin-repository>`
-        ),
-    {
-      'Content-Type': 'text/xml;charset=UTF-8',
-    }
```

Comment:
Do you think there'd be any benefit in leaving a mocked test that works with the legacy api and the xml response? Know we have the live one above, but wanted to ask since that was the case as well before (with all xml/legacy) and this mocked test was still added anyawy

Reply:
done in https://github.com/badges/shields/pull/5974/commits/3f8070d6f337f451849c94c0e350c1b1a37d46ab